native/c/zowex.cpp

```diff
@@ -2329,10 +2329,18 @@ int handle_job_submit_jcl(const ParseResult &result)
     data = zut_encode(data, "UTF-8", string(encoding_opts.codepage), zjb.diag);
   }
 
-  string jobid;
+  int rc = 0;
   rc = zjb_submit(&zjb, data, jobid);
+  if (rc != RTNCD_SUCCESS)
+  {
+    cerr << "Error: could not submit job: '" << jobid << "' rc: '" << rc << "'" << endl;
+    cerr << "  Details: " << zjb.diag.e_msg << endl;
+    return rc;
+  }
 
```

Comment:
We might need the common routine back instead. It can handle returning either the jobid, correlates, or writing a message that the job was submitted. I think the various flags are supported on all of the submit flavors